_non-cosmos/ethereum/assetlist.json

```diff
@@ -1096,6 +1096,37 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/occamfi.png"
         }
       ]
+    },
+    {
+      "description": "The PAGE token is used for actions in the PageDAO NFT literary ecosystem and for DAO governance.",
+      "type_asset": "erc20",
+      "address": "0x60e683C6514Edd5F758A55b6f393BeBBAfaA8d5e",
+      "denom_units": [
+        {
+          "denom": "0x60e683C6514Edd5F758A55b6f393BeBBAfaA8d5e",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "PAGE",
+          "exponent": 8
+        }
+      ],
+      "base": "0x60e683C6514Edd5F758A55b6f393BeBBAfaA8d5e",
+      "name": "Page",
+      "display": "Page",
+      "symbol": "PAGE",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/page.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/page.svg"
+      },
+      "coingecko_id": "page",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/page.png",
```

Comment:
Can you please make the png file have square aspect ratio? (Similar to how the svg already is)